.github/workflows/examples.yml

```diff
@@ -1,6 +1,12 @@
 name: Examples
 
 on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
This reverts a change in #1822 that probably should not have gone into the final merge

If this PR hits any complexity, we can just `git cherry-pick 22cda2daf6e6a369844d2f216d523c70193383c0` to isolate this change

Reply:
Good catch, thanks!